lib/acts_as_scrubbable/scrub.rb

```diff
@@ -1,23 +1,29 @@
 module ActsAsScrubbable
   module Scrub
-
     def scrub!
       return unless self.class.scrubbable?
 
       run_callbacks(:scrub) do
+        if self.class.sterilizable?
+          self.destroy!
```

Comment:
think `delete` is safer, since the other updates also don't run callbacks, but now that i think about it maybe we do want them for destroy in case of `dependent: :destroy`s? dunno...

Reply:
also worried that calling `destroy!` on every row in a table will be pretty slow. can you think of a way to truncate it instead via SQL a-la the after-hook eample?

Reply:
in the `tasks.rb:69` there's an initial check for sterilizable models that will call `delete_all`. you are both correct that it avoids the destroy calls and that there are dependent destroy issues. had realized it running the migration, brought it up in the story. feel like we kind of need the destroy_all b/c of the dependent destroy. side note this also creates an issue for large tables (thinking weight table of 70M records :( )